app/components/event-table.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  didInsertElement() {
+    this.$('.ui.dropdown').dropdown();
```

Comment:
Call the super method before doing anything else here. 
`this._super.call(this)`